app/src/main/res/values/dimens.xml

```diff
@@ -29,5 +29,6 @@
     <dimen name="margin_very_small">2dp</dimen>
     <dimen name="margin_large">10dp</dimen>
     <dimen name="message_textview_bottom_padding">3dp</dimen>
+    <dimen name="textview_layout_max_width">240dp</dimen>
```

Comment:
One sec, sorry, you can close this PR since after adding loop to display result of all action types independently as a new message, this issue will be solved. Pr #681